test/e2e/node/pod_resize.go

```diff
@@ -77,22 +77,22 @@ func doPodResizeResourceQuotaTests(f *framework.Framework) {
 	})
 
 	ginkgo.DescribeTable("pod-resize-resource-quota-test",
-		func(ctx context.Context, desiredContainers []podresize.ResizableContainerInfo, expectedContainers []podresize.ResizableContainerInfo, wantError string) {
+		func(ctx context.Context, desiredContainers []podresize.ResizableContainerInfo, expectedContainers []podresize.ResizableContainerInfo, podResources *v1.ResourceRequirements, wantError string) {
```

Comment:
looks like `podResources` is nil in every test case, can we just remove the parameter?

Reply:
Yes, need to cleanup. I ended up creating a separate test file altogether so this is N/A now.